### DISCUSSION TITLE
Cache bursting mechanism

### DISCUSSION THREAD

**Discussion by savvasshyamsantra** [module-federation/module-federation-examples#920] [Q&A] [answered]
Hi,

Suppose I have one main dashboard, which loads 2 other mfes

dashboard webpack config
 plugins: [
    new ModuleFederationPlugin({
      // For hosts (please adjust)
      remotes: {
        "mfe1": "mfe1@https://abc.com/remoteEntry.js",
        "mfe2": "mfe2@https://xyz.com/remoteEntry.js",
      },
Suppose mfe1 or mfe2 deployed, how the dashboard mfe will get the newer version or mfe1 or mfe2?

**Comment by XiaofengXie16** [comment, marked as answer]
You can append a random number /timestamp at the end . Look at the example in this issue below.
https://github.com/module-federation/module-federation-examples/issues/566
